Containers/clamav/start.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+
+if ! freshclam --foreground --stdout; then
+  exit 1
+fi
```

Comment:
this could also run inside the Dockerfile, which would make the docker image bigger, but the container may starts faster depending on the internet speed it has, but the pull would take longer.
This is needed to download the initial databases which will be updated by the freshclam deamon while running.

Reply:
This is a very good idea! Then we don't need an external internet connection for clamav :)

Reply:
I mean to run it in the Dockerfile already :)